playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerWrapper.java

```diff
@@ -425,12 +431,14 @@ public void onMetadata(Metadata metadata) {
     @Override
     public void load(PKMediaSourceConfig mediaSourceConfig) {
         log.d("load");
-        crossProtocolRedirectEnabled = mediaSourceConfig.crossProtocolRedirectEnabled;
+        crossProtocolRedirectEnabled = mediaSourceConfig.playerSettings.crossProtocolRedirectEnabled();
+        httpDataSourceRequestParams = (mediaSourceConfig.playerSettings.getLicenseRequestAdapter() != null) ? mediaSourceConfig.playerSettings.getLicenseRequestAdapter().adapt(new PKRequestParams(null, new HashMap<String, String>())) : null;
```

Comment:
Please break this line - first call getLicenseRequestAdapter(), then check if it's null, etc.

Reply:
please recheck